x-pack/plugin/eql/qa/rest/src/javaRestTest/java/org/elasticsearch/xpack/eql/EqlRestValidationIT.java

```diff
@@ -31,6 +31,8 @@ protected void assertErrorMessageWhenAllowNoIndicesIsFalse(String reqParameter)
         // https://github.com/elastic/elasticsearch/issues/64197
         // is closed
         assertErrorMessage("inexistent1,inexistent2", reqParameter, """
-            "root_cause":[{"type":"index_not_found_exception","reason":"no such index [null]\"""");
+            "root_cause":[{"type":"index_not_found_exception","reason":"no such index [null]",
+            "resource.type":"index_expression","resource.id":["inexistent1","inexistent2"]}]
+            """);
```

Comment:
Core now throws the "not found" exception, rather than the security code, and it does it in a peculiar way:
https://github.com/elastic/elasticsearch/blob/bbb557b2a15f6340c74644785c6f78cf149e6977/server/src/main/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolver.java#L453